tui.py

```diff
@@ -1,232 +0,0 @@
-import collections
```

Comment:
I guess I can move on knowing tui.py went out at its peak and being forever preserved in glorious git history, long live tui.py!!!